### PR TITLE
Apple landing page clone

### PR DESCRIPTION
**Pull request by thedekerone**
**In this branch i:**
- created the HTML structure
- added the navbar and styled it with flex
- created the main section and styled it with grid
- created the footer section and styled with float

Everything was done with pure HTML and CSS.
Font awesome was used for the icons.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Apple Landing page clone
+
+> this is a clone of the Apple 2014 landing page. The goal was to make it look as similar as possible.
+
```
Comment (majovanilla) [inline README.md:4]: Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...Check this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more info.

**Review by majovanilla — COMMENTED** [review]
Hey @thedekerone , 👋

## You did an *amazing*🌟 ✨ job on this project! 👊🏻  💯

Your page looks _almost_ exactly as the original one!!
You are using semantic HTML.
Your pull request message is great.
You are using a good Readme template.
Some suggestions for the next project: Commit more often so that we can see all your progress.

There are still some changes 👇 you need to implement to approve this request 🔍

### Required Changes: ✍️

- Always remember to activate your repo in Stickler. I cannot see your linters working!!
- Check the comment on the Readme and improve it. 

### Optional Changes:

If you want to make your project even better 🔥💥, I have a few suggestions you can implement.  

These are completely [Optional]: 🦔

- Your project would look sooo much better if you include the hover effects on the links of the page, like the original version. This includes the ones on the footer and the navbar.

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@majo)
I would be happy to help! 👩‍💻 


Happy coding 🤓

--Maria Reyes

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — APPROVED** [review]
## Status: Approved 🏆 🎈 

Hi @thedekerone, you did a great job fixing all the comments pointed out by the previous TSE 👍 I hereby approve of your project 🙂 Go ahead and merge to master and submit the project completion form 👍 

Good luck with your next projects 🚀 
_Happy coding_ 👨‍💻 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**